src/routes/home/Home.js

```diff
@@ -163,7 +164,7 @@ async function extractDate({ attachmentFile, attachmentArrayBuffer, ext }) {
       millisecondsSinceEpoch: CreateDate.getTime(),
       offset: OffsetTimeDigitized
         ? parseInt(OffsetTimeDigitized, 10) * -60
-        : new Date().getTimezoneOffset(),
+        : getNycTimezoneOffset(CreateDate),
```

Comment:
We might want to keep the `new Date().getTimezoneOffset()` for when `CreateDate` doesn't exist. Probably can do it like this:

```suggestion
        : getNycTimezoneOffset(CreateDate || new Date),
```

or like this:

```suggestion
				: CreateDate
        	? getNycTimezoneOffset(CreateDate),
        	: new Date().getTimezoneOffset(),
```